File: src/webs/CSSPWebToolsAng/client-apps/src/app/models/webhydrometricdatavalue.ts
import { HydrometricDataValue } from './generated/HydrometricDataValue.model';
import { WebBase } from './webbase';

export class WebHydrometricDataValue {
    TVItemParentList: WebBase[] = [];
    HydrometricDataValueList: HydrometricDataValue[] = [];
}

File: src/services/CSSPModels_backup/srcweb/WebHydrometricDataValue.cs
/*
 * Manually edited
 * 
 */
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace CSSPModels
{
    [NotMapped]
    public partial class WebHydrometricDataValue
    {
        #region Properties
        public List<WebBase> TVItemParentList { get; set; }
        public List<HydrometricDataValue> HydrometricDataValueList { get; set; }
        #endregion Properties

        #region Constructors
        public WebHydrometricDataValue()
        {
            TVItemParentList = new List<WebBase>();
            HydrometricDataValueList = new List<HydrometricDataValue>();
        }
        #endregion Constructors
    }
}
